Core/Src/housekeeping.c

```diff
@@ -19,6 +19,7 @@ housekeeping_packet_t _get_housekeeping() {
     hk.gate_temp = get_temp(GATE_TEMP_SENSOR);
     hk.imagenum = get_image_num(1);
     hk.software_version = software_ver;
+    hk.errornum = 0x00;
```

Comment:
Can't have any errors if the error number is hardcoded to 0 amiright

Reply:
Yes, although I was not sure where was the error counter

Reply:
I haven't implemented it yet. I'll implement it alongside our audit log

Reply:
sounds good